lib/admin-page.php

```diff
@@ -607,8 +607,10 @@ function classicpress_check_can_migrate() {
 	}
 
 	// Check: Supported PHP version
-	$php_version_min = '5.6';
-	if ( version_compare( PHP_VERSION, $php_version_min, 'lt' ) ) {
+	if ( isset( $parameters['php'] ) &&
+		version_compare( PHP_VERSION, $parameters['php']['min'], 'lt' ) ||
+		version_compare( PHP_VERSION, $parameters['php']['max'], 'gt' )
+	) {
```

Comment:
How about this instead, making the check fail if the php parameters are somehow not set, and also simplifying the logic a bit:

```suggestion
	if (
		! isset( $parameters['php'] ) ||
		version_compare( PHP_VERSION, $parameters['php']['min'], 'lt' ) ||
		version_compare( PHP_VERSION, $parameters['php']['max'], 'gt' )
	) {
```

Reply:
Actually that's not great either because later code will try to read `$parameters['php']['min']` etc if `$parameters['php']` is not set.

I'm not sure the best way to handle this. Maybe just leave off the `isset` check entirely since that should never happen?

Reply:
In this PR and in the changes I made to the theme and plugin checks I added `isset()` checks to aid me during development while the API endpoint is not updated.

I would presume there is a good argument to taking this instance and all of the other out if we are sure the API is correct.

Reply:
Ok, let's just leave it as you had it for now but fix the indentation. I think I am nitpicking too much on that anyway.